content/docs/how-to-guides/warehouses/bigquery.md

```diff
@@ -29,3 +29,16 @@ CREATE OR REPLACE TABLE dataform.example
 PARTITION BY DATE(ts)
 AS (SELECT CURRENT_TIMESTAMP() AS ts)
 ```
+
+If desired, partitions can be clustered by using the `clusterBy` option, for example:
+
+```js
+config {
+  type: "table",
+  bigquery: {
+    partitionBy: "DATE(ts)",
+    clusterBy: ["name", "revenue"]
+  }
+}
+SELECT CURRENT_TIMESTAMP() AS ts
```

Comment:
Can you add a field such as "name" and use it in the clusterBy, so that this whole example works if you copy paste it?

Reply:
Oops, nice catch